packages/components/src/components/InteractiveOverlay/InteractiveOverlay.tsx

```diff
@@ -7,11 +7,13 @@ const StyledItem = styled.a`
   border: none;
   padding: 0;
   background: transparent;
+  display: flex;
```

Comment:
lazy solution, gets rid of the extra line height when the anchor tag is default (inline)